<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutOuvrageInterface</class>
 <widget class="QDialog" name="AjoutOuvrageInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AjoutOuvrageInterface</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 italic 14pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Auteur(s) </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 italic 14pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Titre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 italic 14pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Éditeur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 italic 14pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Ville d'édition</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 italic 14pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Année d'édition</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>430</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 italic 14pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Code ISBN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman [UKWN]</family>
     <pointsize>16</pointsize>
     <weight>7</weight>
     <italic>true</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 63 italic 16pt &quot;URW Bookman [UKWN]&quot;;</string>
   </property>
   <property name="text">
    <string>Formulaire d'ajout de l'ouvrage</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="auteurTxt">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="titreTxt">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="villeEditionTxt">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editeurTxt">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="codeIsbnTxt">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>500</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 127);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="anneeEditionSpn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>451</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>AjoutOuvrageInterface</receiver>
   <slot>validationSaisieFormulaire()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validationSaisieFormulaire()</slot>
 </slots>
</ui>
